pennylane/qchem/hartree_fock.py

```diff
@@ -210,10 +213,10 @@ def _nuclear_energy(*args):
             coor = args[0]
         else:
             coor = r
-        e = anp.array([0.0])
+        e = np.array([0.0])
```

Comment:
Here I think qml.math.array should be fine as well

Reply:
Done.